test/index.html

```diff
@@ -22,7 +21,8 @@
 			'./skeleton-test.html',
 			'./entity.html',
 			'./mixin/entity-mixin-test.html',
-			'./EnrollmentEntity/EnrollmentEntity.html'
+			'./EnrollmentEntity/EnrollmentEntity.html',
+			'./SequenceEntity/SequenceEntity.html'
```

Comment:
Just a heads up that we may have a conflict here. https://github.com/BrightspaceHypermediaComponents/siren-sdk/pull/20